Marlin/Configuration.h

```diff
@@ -111,15 +111,15 @@
  *
  * :[2400, 9600, 19200, 38400, 57600, 115200, 250000, 500000, 1000000]
  */
-#define BAUDRATE 250000
+#define BAUDRATE 115200
```

Comment:
You no change default config. Change default config bad.

Reply:
Noted.  Was not sure how this worked. Thanks for the review and fixing.